presto-native-execution/scripts/release-centos-dockerfile/Dockerfile

```diff
@@ -45,18 +48,20 @@ ENV PRESTO_ENABLE_HDFS=ON
 ENV PRESTO_ENABLE_TESTING=ON
 ENV CLEANUP_BUILD_ARTIFACTS=1
 
-SHELL ["/bin/bash", "-c"]
+SHELL ["/bin/bash", "-exu", "-c"]
 #-----------------------------------------------------------
 #---( START of PrestoDB and Velox dependency tree build scripts
 #-----------------------------------------------------------
 WORKDIR ${PRESTODB_HOME}
+COPY --chown=root * "${PRESTODB_HOME}/_cicd"
 RUN --mount=type=ssh \
-    set -exu && \
     dnf -y install openssh-clients git wget unzip make && \
     mkdir -p -m 0600 /root/.ssh && \
     ! ssh-keyscan github.com >> /root/.ssh/known_hosts && \
+    [ $(ls "${PRESTODB_HOME}/_cicd" | wc -l) -ge 10 ] && \
+    mv "${PRESTODB_HOME}/_cicd" "${PRESTODB_HOME}/_repo" || ( \
     git clone --progress ${PRESTODB_REPOSITORY} "${PRESTODB_HOME}/_repo" && \
```

Comment:
git clone/checkout is not needed in a CI pipeline.

Reply:
I will add comment to clarify this - the git clone and git checkout is not called in case of Docker context being the root of presto repository